lighthouse-core/test/audits/metrics/first-meaningful-paint-test.js

```diff
@@ -22,7 +22,7 @@ describe('Performance: first-meaningful-paint audit', () => {
     const context = {options, settings: {throttlingMethod: 'provided'}, computedCache: new Map()};
     const fmpResult = await FMPAudit.audit(artifacts, context);
 
-    assert.equal(fmpResult.score, 1);
+    assert.equal(fmpResult.score, 0.96);
```

Comment:
kinda weird to see all these scores change in the tests.

im assuming thats because `TestedAsMobileDevice` is falsy by default in these tests.
that's a little odd as LH is is otherwise mobile by default.

we need something better here. options i can think of:

- add `{TestedAsMobileDevice: true}` into all these tests to make it explicit
- change the ternary in the metric audits to check if `TestedAsMobileDevice === false ? 'desktop' : 'mobile'`  because i guess it'd be `undefined` in all these test scenarios?

Reply:
I considered both of those.

the first was annoying and a big change to the tests

the second is ehhhh. I would do this if only there was one single place this condition lived. but it's in many places

but
> that's a little odd as LH is is otherwise mobile by default.
is so convincing that we should do one of these. I choose the second. is there a good place to move this code to a function?

Reply:
ehhhhh i just did 2)